graphql-ballerina/records.bal

```diff
@@ -39,58 +39,32 @@ public type RequestLimitConfigs record {|
     *http:RequestLimitConfigs;
 |};
 
-# Represents the data in an output object for a GraphQL query.
-public type Data record {
+type Data record {
     // Intentionally kept empty
```

Comment:
```suggestion
    // Intentionally, kept empty.
```

Reply:
This is a dev comment. Not a doc comment